product_form_sale_link/views/product_template.xml

```diff
@@ -5,7 +5,6 @@
     <record model="ir.ui.view" id="product_template_form_view">
         <field name="name">product.template.form (in product_form_sale_link)</field>
         <field name="model">product.template</field>
-        <field name="groups_id" eval="[(4, ref('sales_team.group_sale_salesman'))]" />
```

Comment:
This change is not valid. for people that are not member of group_sale_salesman, the field sale_lines_count will be displayed, but the button action_product_template_sale_list will be displayed.

Reply:
@legalsylvain Since v16, groups are not allowed anymore on views. The field is in button, so it won't be displayed.